build_test/test/package_reader_test.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:build/build.dart';
+import 'package:build_test/build_test.dart';
+import 'package:glob/glob.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('$PackageAssetReader', () {
+    AssetReader reader;
+
+    final buildTest = new AssetId('build_test', 'lib/build_test.dart');
+    final buildMissing = new AssetId('build_test', 'lib/build_missing.dart');
+    final thisFile = new AssetId('build_test', 'test/package_reader_test.dart');
+
+    setUp(() async {
+      reader = await PackageAssetReader.currentIsolate(
+        rootPackage: 'build_test',
```

Comment:
Yeah I assumed we could figure it out, but seemed easy enough to define it for now and we can always fill in if null in the future.